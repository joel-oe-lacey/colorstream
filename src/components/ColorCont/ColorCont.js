import React from 'react';
import "./ColorCont.scss";
import ColorBox from '../ColorBox/ColorBox';

const ColorCont = () => {
    return (
        <section className="ColorDisp">
            <ColorBox />
            <ColorBox />
            <ColorBox />
            <ColorBox />
        </section>
    )
}

export default ColorCont;