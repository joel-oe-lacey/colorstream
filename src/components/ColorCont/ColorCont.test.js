import React from 'react';
import { shallow } from 'enzyme';
import ColorCont from './ColorCont';

describe('ColorCont', () => {
    let wrapper;
    const mockColors = ['B8C480', 'D4E79E', '922D50', '501537']

    beforeEach(() => {
        wrapper = shallow(<ColorCont colors={mockColors}/>);
    });

    it('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})