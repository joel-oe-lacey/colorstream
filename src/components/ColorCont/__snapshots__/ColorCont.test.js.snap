// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorCont should match the snapshot 1`] = `
<section
  className="ColorDisp"
>
  <ColorBox
    color="B8C480"
    key="0"
  />
  <ColorBox
    color="D4E79E"
    key="1"
  />
  <ColorBox
    color="922D50"
    key="2"
  />
  <ColorBox
    color="501537"
    key="3"
  />
</section>
`;
